Knossos.NET/Models/GlobalSettings.cs

```diff
@@ -43,6 +43,22 @@ public enum Decompressor
     /// </summary>
     public class GlobalSettings
     {
+        /// <summary>
+        /// Flags handled by globalsettings, any flag that is present in this array
+        /// will be ignored in mod cmdline if it exist.
```

Comment:
"if present." is a little better than "if it exist."

Reply:
ok